CONTRIBUTING.md

```diff
@@ -47,8 +47,8 @@ Rake can be found at http://docs.seattlerb.org/rake/.
 
 To run a single test suite
 
-   rake test TEST=path/to/test.rb
+`rake test TEST=path/to/test.rb`
```

Comment:
I took a look in all bash code that we have at .md before answer and saw that they always have $.